contracts/Example3.sol

```diff
@@ -1,14 +1,18 @@
 pragma solidity ^0.8.4;
 
+// uint256 marginPercentage not needed
+// Not public, no getter/setter, it's just a constant.
+// Replace with '30'
+
 contract Example3 {
-    uint256 marginPercentage = 30;
+    //uint256 marginPercentage = 30;
     function getOwnerMargin(uint256 amount)
         public
         view
         returns (uint256 amountForSender, uint256 amountForOwner)
     {
-        amountForSender = (amount * (100 - marginPercentage)) / 100;
-        amountForOwner = (amount * marginPercentage) / 100;
+        amountForSender = (amount * (100 - 30)) / 100;
```

Comment:
Best to have marginPercentage stored in a variable as it is being accessed twice. To avoid accessing storage twice you can define it inside the function itself

Reply:
Thanks!